espnet/asr/pytorch_backend/asr.py

```diff
@@ -167,8 +168,10 @@ def update_core(self):
 
         # Get the next batch ( a list of json files)
         batch = train_iter.next()
-        # self.iteration += 1 # Increase may result in early report, which is done in other place automatically.
-        x = tuple(arr.to(self.device) for arr in batch)
+        self.iteration += 1
```

Comment:
could you tell me why you revert `self.iteration += 1`?

Reply:
This is my mistake.

Reply:
Maybe I uncommented it during fixing some conflicts by mistake.